### DISCUSSION TITLE
Cannot assign requested address - bind(2) for "xx.xxx.xx.xx" port

### DISCUSSION THREAD

**Discussion by 0nurkaplan** [postalserver/postal#2409] [Help with using Postal]
Hello,

Recently we are getting the following error on gmail side

Cannot assign requested address - bind(2) for "xx.xxx.xx.xx" port

what is the reason for this

Thanks..

**Comment by willpower232** [comment]
It sounds like you have something already on that port maybe? You can check what is on a port with `netstat -lnp`.

Share a screenshot if you're still stuck.

**Comment by 0nurkaplan** [comment]
We have cluster postal  installation on  two servers. Each  server has four IP addresses.  We found that, error caused by postal trying to bind other servers IP addresses. @willpower232

**Reply by willpower232** [reply]
are you trying to use IP Pools to manage or specify which IP addresses are used?

**Reply by 0nurkaplan** [reply]
We have created a repository for the brand, but we are getting a header error on some ip addresses.

Cannot assign requested address - bind(2) for "xx.xxx.xx.xx" port

**Reply by 0nurkaplan** [reply]
@willpower232 Yes, we are using IP pools.

**Reply by willpower232** [reply]
And each server has different IP addresses so each server has only some or none of the IP addresses in the pool?

Does Postal actually start or does the smtp bit refuse to since it can't find all the IP addresses it is thinking of?

I'm not sure there is a way around this as I can imagine the queue of emails to be sent might not have any context as to what IP address it should be sent from until its too late.

**Reply by 0nurkaplan** [reply]
the error we got is only two ip addresses, the others are ok 

Cannot assign requested address - bind(2) for "xx.xxx.xx.xx" port
